doc/releases/quincy.rst

```diff
@@ -5,6 +5,114 @@ Quincy
 Quincy is the 17th stable release of Ceph.  It is named after Squidward
 Quincy Tentacles from Spongebob Squarepants.
 
+v17.2.1 Quincy
+==============
+
+This is the second stable release of Ceph Quincy.
+
+Notable Changes
+---------------
+* The "BlueStore zero block detection" feature (first introduced to Quincy in
+https://github.com/ceph/ceph/pull/43337) has been turned off by default with a
+new global configuration called `bluestore_zero_block_detection`. This feature,
```

Comment:
s/configuration/option/

We seem to mostly refer to tunables as "options"

Reply:
There is also a `Bullet list ends without a blank line; unexpected unindent.` sphinx warning here -- the body needs to be indented.